Snakefile

```diff
@@ -60,6 +60,21 @@ rule prepare_conv_pp:
     shell:
         "python scripts/prepare_conv_pp.py {input.opsd} {input.gpkg} {input.b3_regions} {input.scalar_template} {output}"
 
+
+rule prepare_heat_demand:
+    input:
+        raw="raw/",
+        dishh="raw/distribution_households.csv",
+        hl="raw/holidays.csv",
+        script="scripts/prepare_heat_demand.py",
+        sc="raw/scalars.csv",
+        tstemp="oemof_b3/schema/timeseries.csv"
+    output:
+        "results/_resources/heat_load.csv"
```

Comment:
```suggestion
        "results/_resources/load_profile_heat.csv"
```

Reply:
To parallel the name of the electricity profiles.

Reply:
Done with commits https://github.com/rl-institut/oemof-B3/pull/76/commits/f2b37f8082de1b48b5c269947c5a8171703cf94b and https://github.com/rl-institut/oemof-B3/pull/76/commits/cf677c1b9cb7e06da99baebc17180e89fe89465f.